docker-compose-override.yml

```diff
@@ -30,7 +30,7 @@ services:
       - TEMPLATES_AUTO_RELOAD=True
     command: bash -c "/content/scripts/start-server-debug.sh"
     restart: "unless-stopped"
-    image: registry.cern.ch/cernopendata/cernopendata-portal:0.2.5
+    image: registry.cern.ch/cernopendata/cernopendata-portal:0.2.9
```

Comment:
```suggestion
    image: registry.cern.ch/cernopendata/cernopendata-portal:prod
```

Reply:
Perhaps there is a container image tag called `prod`, but there is no such git tag in the `cernopendata-portal` repository. That could be problematic; recall that we have been using `latest` in the past, and had to stop it due to local installation being broken. This is what lead to using 0.1.11 docker image in the sources here, with occasional update to later stable versions.

I therefore prefer to use concrete versions (which exist as both git tags and docker image versions, and which are immutable) over something like `latest` or `prod` (which is mutable and which does not exist as a tag or a branch).